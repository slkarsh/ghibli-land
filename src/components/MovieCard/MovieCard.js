import React from 'react'


export const MovieCard = ({ title, description, release_date, rt_score}) => {
  return (
    <section>
      <h3>{title}</h3>
      <p className='description'>{description}</p>
      <p className='date'>{release_date}</p>
      <p className='score'>{rt_score}</p>
    </section>
  )
}

export default MovieCard