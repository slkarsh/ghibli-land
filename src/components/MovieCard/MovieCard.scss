img {
  height: 60%;
  width: 100%;
}

// .movie-card {
//     border-radius: 6px 6px 6px 6px;
//     background: linear-gradient(to bottom, rgb(85, 85, 85), #2d2d2d00);
//     display: flex;
//     border: solid rgb(143, 143, 143) 1px;
//     box-shadow: 0 0 3px rgb(130, 130, 130);
//     flex-direction: column;
//     align-items: flex-start;
//     justify-content: space-between;
//     transition: .2s;
//     margin: 10px 50px;
// }

.movie-card {
  border: 2px solid black;
}