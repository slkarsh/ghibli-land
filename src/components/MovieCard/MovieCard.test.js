import { MovieCard } from './MovieCard'
import React from 'react'
import { shallow } from 'enzyme'

describe('MovieCard', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<MovieCard
        title='a film'
        description='a plot'
        release_date='2001'
        rt_score={89}
        setId={2}
       />)

       expect(wrapper).toMatchSnapshot()
  })
})