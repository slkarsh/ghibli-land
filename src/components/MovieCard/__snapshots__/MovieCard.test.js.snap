// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard should match the snapshot 1`] = `
<section
  className="movie-card"
>
  <img
    alt="movie poster"
    className="poster-image"
    src="https://image.tmdb.org/t/p/w1280/rtGDOeG9LzoerkDGZF9dnVeLppL.jpg"
  />
  <h3>
    a film
  </h3>
  <p
    className="description"
  >
    a plot
  </p>
  <p
    className="date"
  >
    Released in: 
    2001
  </p>
  <p
    className="score"
  >
    Rotten Tomatoes: 
    89
    %
  </p>
</section>
`;
